src/Models/Backup.php

```diff
@@ -177,16 +174,26 @@ public function restore()
      */
     public function serialize()
     {
-        $date_format = $this->getConfig()->get('date_format');
         return [
             'file'      => $this->get('filename'),
             'size'      => $this->getSizeInMb(),
-            'date'      => date($date_format, $this->getDate()),
-            'expiry'    => date($date_format, $this->getExpiry()),
+            'date'      => $this->getConfig()->formatDatetime($this->getDate()),
+            'expiry'    => $this->getConfig()->formatDatetime($this->getExpiry()),
```

Comment:
Should this just be `$this->getDate()` + `$this->getExpiry()`, now that you are using table formatters?

`Metric` returns an unformatted date in serialize, as does `Site` and `Workflow`.
`NewRelic` returns a formatted date.

It would be good to make the default date formats consistent, e.g. seconds since epoch for machine formats.

Reply:
Yes, it should be. I missed this.